cpp/include/Ice/LocalException.h

```diff
@@ -25,21 +25,173 @@
 namespace Ice
 {
     /**
-     * Abstract base class for all Ice run-time exceptions.
+     * This exception is raised if a request failed. This exception, and all exceptions derived from
+     * {@link RequestFailedException}, are transmitted by the Ice protocol, even though they are declared
+     * <code>local</code>.
      * \headerfile Ice/Ice.h
      */
-    class ICE_API LocalException : public Exception
```

Comment:
I want to split this file in LocalException.h and LocalExceptions.h in a follow-up PR.